tokio-util/CHANGELOG.md

```diff
@@ -1,3 +1,25 @@
+# 0.7.4 (September 7, 2022)
+
+### Added
+
+- io: add `SyncIoBridge::shutdown()` ([#4938])
+- task: improve `LocalPoolHandle` ([#4680])
+
+### Fixed
+
+- util: add `track_caller` to public APIs ([#4785])
+
+### Unstable
+
+- task: fix compilation errors in `JoinMap` with Tokio v1.21.0 ([#4755])
+- task: remove the unstable, deprecated `JoinMap::join_one` ([#4920])
```

Comment:
I was going to ask why `JoinMap` wasn't stabilized in this release, since `JoinSet` stabilized in v1.21.0, but then I remembered `JoinMap` also depends on task IDs, which are still unstable :+1: